CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing guidelines
+---
+
+We're always open to contributions of any size! But before you start, read through our guidelines so your PR can be merged quickly. 
+
+## How to contribute:
+
+1. If this is your first time here, check out our issues page and filter for ["good first issue"](https://github.com/superwise-ai/elemeta/labels/good%20first%20issue) for inspiration on where to get started.
+2. Fork the repository.
+3. Make your changes in your local branch.
+4. Follow our repo [guidelines](#guidelines).
+5. Check to make sure that your commits follow the package standards and then open a new PR request to the "main" branch.
+
+---
+
+## <a id="guidelines"></a>Guidelines:
+
+### Formatting
+We use isort and black formatters in Elemeta. To apply these formats automatically, please run the following command from the repo root 
+```sh
```

Comment:
@BigicecreamTaken Why not merging the two scripts: `scripts/linters.sh` and `scripts/formatters.sh`?

Reply:
Because CI/CD run the `linters.sh` before approveing the PR
maybe we can make `formatters.sh` run `linters.sh` after formating(if we do so I think we should change the script name to something else)

Reply:
I think we should do that. @roeizavida opinion?

Reply:
We should create a `pre-commit` config that will run the required steps and also document the process for running manually. Let's open a new issue for it, as it would be out of scope for this PR.